hdf5.spec

```diff
@@ -583,7 +585,7 @@ done
 %endif
 
 %changelog
-* Wed Jul 22 2020 Maureen Jean <maureen.jean@intel.com> - 1.12.0-1.g07066a381e
+* Wed Jul 22 2020 Maureen Jean <maureen.jean@intel.com> - 1.12.0-1
 - Update HDF5 to version 1.12.0
 
```

Comment:
Let's add some more of the change details here, like the creation of mpich and openmpi3 variants, how those obsolete the previous non-varianted packages, etc.